.travis.yml

```diff
@@ -1,6 +1,9 @@
 language: python
 sudo: false
-cache: pip
+cache:
+  directories:
+    - $HOME/.cache/pip
+    - $HOME/virtualenv
```

Comment:
sorry you're totally right. i think i lost my train of thought when researching that yesterday